AppCenter/AppCenter/Internals/Sender/MSOneCollectorIngestion.m

```diff
@@ -0,0 +1,103 @@
+#import "MSAppCenterInternal.h"
+#import "MSAppCenterErrors.h"
+#import "MSConstants+Internal.h"
+#import "MSHttpSenderPrivate.h"
+#import "MSLog.h"
+#import "MSLogContainer.h"
+#import "MSLoggerInternal.h"
+#import "MSOneCollectorIngestion.h"
+#import "MSUtility+Date.h"
+
+NSString *const kMSOneCollectorApiVersion = @"1.0";
+NSString *const kMSOneCollectorApiPath = @"/OneCollector";
+NSString *const kMSOneCollectorContentType = @"application/x-json-stream; charset=utf-8;";
+NSString *const kMSOneCollectorApiKey = @"apikey";
+NSString *const kMSOneCollectorClientVersionKey = @"Client-Version";
+NSString *const kMSOneCollectorUploadTimeKey = @"Upload-Time";
+
+@implementation MSOneCollectorIngestion
```

Comment:
I have to admit, I am not confortable with having that much duplicate code between MSAppCenterIngestion and this one but I guess it will diverge much more in the future while we'll bring more features to this one.

Reply:
Valid point but we will eventually remove App Center Ingestion, right? We can avoid to revert changes at that time and also align to Android implementation.

Reply:
This was a conscious decision by @jaelim-ms  and myself and aligns with the decision that was made for the Android SDK.

Reply:
The "eventually" is in a big while to me but fine.